### PR TITLE
remove _t suffix from types

### PR DESCRIPTION
**Pull request by mat-hek**
Used the following regex (in VS Code) :D
from `_t([^a-z]|$)`
to `$1`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 ## 1.0.0
```
Comment (varsill) [inline CHANGELOG.md:3]: I would add a point saying that we have renamed types there

File: guides/upgrading/v0.11.md [hunk 2]
```diff
@@ -292,7 +292,7 @@ Rename `Membrane.Time.to_<unit name>/1` into `Membrane.Time.round_to_<unit name>
 ```
 
 ## Update the children definitions
-Children defintions syntax (previously known as `ParentSpec`, after the name of a structure used to define children), that was used in `Membrane.Pipeline.Action.spec_t` and `Membrane.Bin.Action.spec_t` actions, has changed. 
+Children defintions syntax (previously known as `ParentSpec`, after the name of a structure used to define children), that was used in `Membrane.Pipeline.Action.spec` and `Membrane.Bin.Action.spec` actions, has changed.
```
Comment (varsill) [inline guides/upgrading/v0.11.md:295]: I am not sure we should change it there

File: lib/membrane/core/pipeline/zombie.ex [hunk 3]
```diff
@@ -1,6 +1,6 @@
 defmodule Membrane.Core.Pipeline.Zombie do
   @moduledoc false
-  # When a pipeline returns Membrane.Pipeline.Action.terminate_t()
+  # When a pipeline returns Membrane.Pipeline.Action.terminate()
   # and becomes a zombie-ie-ie oh oh oh oh oh oh oh, ay, oh, ya ya
```
Comment (varsill) [inline lib/membrane/core/pipeline/zombie.ex:4]: I wasn't aware that one is still there :D

**Review by varsill — CHANGES_REQUESTED** [review]
We need to preserve that regex for the posterity :D

File: lib/membrane/core/pipeline/zombie.ex (hunk 3 above)
Comment (mat-hek) [reply lib/membrane/core/pipeline/zombie.ex:4]: It was actually added in 0.11 IIRC :P

File: guides/upgrading/v0.11.md (hunk 2 above)
Comment (mat-hek) [reply guides/upgrading/v0.11.md:295]: Why so?

File: lib/membrane/core/pipeline/zombie.ex (hunk 3 above)
Comment (varsill) [reply lib/membrane/core/pipeline/zombie.ex:4]: I know, but I thought it was changed before merging :D

File: guides/upgrading/v0.11.md (hunk 2 above)
Comment (varsill) [reply guides/upgrading/v0.11.md:295]: Well, we are refering to somthing `that WAS used in ...` - back in the time, it was `Membrane.Pipeline.Action.spec_t`, not `Membrane.Pipeline.Action.spec` :D

**Review by varsill — APPROVED** [review]
LGTM 🥇

**Pull request merged**